src/scss/objects/_form-controls.scss

```diff
@@ -0,0 +1,240 @@
+/**
+ * Custom Checkboxes and radios
+ *
+ * --------------------------------
+ *
+ * Checkbox button example HTML
+ *
+ *    <label class="c-form-label" for="checkbox1">
+ *        <input class="c-form-input" type="checkbox" id="checkbox1" name="myCheckbox" value="A value"/>
+ *        <span class="c-form-indicator c-form-indicator--checkbox"></span>
+ *        My Checkbox Label Text
+ *    </label>
+ *
+ * -------------------------------
+ *
+ * Radio button example HTML
+ *
+ *    <label class="c-form-label" for="radio1">
+ *        <input class="c-form-input" type="radio" id="radio1" name="myRadioBtn" value="A value"/>
+ *        <span class="c-form-indicator c-form-indicator--radio"></span>
+ *        My Radio Button Label Text
+ *     </label>
+ *
+ * ================================= */
+ $formControl-bgColor                  : $white !default;
+ $formControl-bgColor-disabled         : $color-disabled;
+ $formControl-borderColor              : $color-border;
+ $formControl-color             : $blue !default;
+ $formControl-background-color  : $color-bg !default;
+ $formControl-hover-color       : $blue--dark !default;
+ $formControl-borderColor-hover : $blue !default;
+ $formControl-margin-left       : 0 !default;
+ $formControl-padding-left      : spacing(x4) !default;
+ $formControl-width--wide       : 20px !default;
+ $formControl-width             : 24px !default;
+ $formControl-height--wide      : 20px !default;
+ $formControl-height            : 24px !default;
+ $formControl-border-width      : 1px;
+
+
+ .o-formControl-label {
+     position: relative;
+     display: block;
+     cursor: pointer;
+     user-select: none;
+     padding: spacing();
+     padding-left: $formControl-padding-left;
+
+     &:hover .o-formControl-indicator {
+         border-color: $formControl-borderColor-hover;
+     }
+
+     &:hover .o-formControl-input:disabled ~ .o-formControl-indicator {
+         border-color: $formControl-borderColor;
+         cursor: not-allowed;
+     }
+
+     .o-formControl-input:disabled ~ .o-formControl-indicator {
+        background-color: $formControl-bgColor-disabled;
+     }
+ }
+
+ // Give inline elements some space to the right
+ .o-formControl-label--inline {
+     display: inline-block;
+     padding-right: 2em;
+ }
+
+     // The hidden input
```

Comment:
Not sure what is meant by this comment? If it's a hidden input shouldn't the selector be named to describe what it is doing? Could also be added as a modifier to `o-formControl-input`?

Reply:
Perhaps not the best description. Its to hide the default radio or checkbox so you only see the label and the designed radio/checkbox